### PR TITLE
tds-winner-tool: Exclude list of public keys from winners

### PR DESCRIPTION
**Pull request by jstarry**

### CODE REVIEW

File: winner-tool/src/main.rs [hunk 1]
```diff
@@ -57,13 +60,14 @@ fn main() {
                 .help("Public key of the baseline validator"),
         )
         .arg(
-            Arg::with_name("bootstrap_leader")
-                .long("bootstrap-leader")
+            Arg::with_name("exclude_keys")
+                .long("exclude-keys")
```
Comment (mvines) [inline winner-tool/src/main.rs]: nit: `exclude-pubkeys` to clarify the kind of "key" please

File: winner-tool/src/main.rs [hunk 2]
```diff
@@ -57,13 +60,14 @@ fn main() {
...
-                .long("bootstrap-leader")
+            Arg::with_name("exclude_pubkeys")
+                .long("exclude-pubkeys")
                 .value_name("PUBKEY")
+                .multiple(true)
                 .takes_value(true)
                 .required(true)
-                .validator(is_pubkey)
-                .help("Public key of the bootstrap leader"),
+                .validator(is_pubkey_list)
```
Comment (CriesofCarrots) [inline winner-tool/src/main.rs]: `clap` applies the validator separately for each value of a `multiple` arg, so this should remain `is_pubkey`
(Here's a silly little demo of this functionality: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=711fcb4e347b9b092fc3c6606c4dadf1)

File: winner-tool/src/main.rs [hunk 3]
```diff
@@ -136,12 +124,12 @@ fn main() {
         override_num_threads: Some(1),
     };
 
-    let excluded_set = {
-        let mut set = HashSet::new();
-        set.insert(baseline_id);
-        set.insert(bootstrap_id);
-        set
-    };
+    let excluded_set = exclude_pubkeys
```
Comment (CriesofCarrots) [inline winner-tool/src/main.rs]: I think you can write this in one line as:
`let excluded_set: HashSet<Pubkey> = exclude_pubkeys.into_iter().collect();`
or
`let excluded_set: HashSet<Pubkey> = HashSet::from_iter(exclude_pubkeys);`
(will have to `use std::iter::FromIterator;` for the latter)

File: winner-tool/src/main.rs (hunk 2 above)
Comment (jstarry) [reply winner-tool/src/main.rs]: Great catch! Thank you

File: winner-tool/src/main.rs (hunk 3 above)
Comment (jstarry) [reply winner-tool/src/main.rs]: Much better, thanks!

**Review by CriesofCarrots — APPROVED** [review]

**Pull request merged**